TESTING.md

```diff
@@ -300,6 +300,7 @@ Test ideas:
 9. Use only the keyboard. For example, navigate transcripts with arrows, delete, enter.
 10. Start typing while being focused on Chat History to perform search-by-title.
 11. Open multiple chats and ask few simultaneous questions in several sessions at once.
+12. Open new chat with <kbd>Alt</kbd> + <kbd>=</kbd> shortcut.
```

Comment:
pls provide macOs shortcut too!

Reply:
Oh, good point!

Can you please confirm that `Option` + `=` opens a new chat?

Reply:
Can you also please confirm that this is good UX from the perspective of a Mac user? (e.g., there's no need to press a third additional button or whether it's consistent with key conventions)

Reply:
it does not work

Reply:
oh, it works but it does not open cody tool window - pls fix it

Reply:
Thanks for checking! That's strange, because for me, the "New Chat" action automatically switches the window with focus on message panel. 

I'll look into this.

Reply:
Update after the call with Mikołaj - the switching works and automatically opens the panel, but if we hide the Cody toolbar, it doesn't show up automatically.

Reply:
@mkondratek, fixed!